FF1Lib/Hacks.cs

```diff
@@ -170,7 +170,7 @@ public void RollCredits()
 			// leading spaces are used to increment the PPU ptr precisely to save ROM space.
 			List<string[]> texts = new List <string[]>();
 			texts.Add(new string[] { "", "",
-				                     " Final Fantasy  ",
+				                     " Final  Fantasy ",

```

Comment:
The extra space in between "centers" the entire string better and looks a lot better in game IMHO.